DQM/TrackingMonitor/interface/TrackAnalyzer.h

```diff
@@ -136,6 +136,21 @@ class TrackAnalyzer
 	    , NumberOfValidRecHitsPerTrackVsEta(NULL)
 	    , NumberOfValidRecHitVsPhiVsEtaPerTrack(NULL)
 
+            , NumberOfLostRecHitsPerTrackVsPhi(NULL)
```

Comment:
me too.
will try.
Still I am almost sure this is not the problem...
maybe is
if(doDCAPlots_ || doPVPlots_ || doSIPPlots_ || doAllPlots_) {
while the booking is only
if(doDCAPlots_ ||  || doAllPlots_) {